app/assets/stylesheets/govuk_publishing_components/govuk_frontend_support.scss

```diff
@@ -2,3 +2,18 @@
 // It provides govuk-frontend but adds no weight to the compiled CSS
 @import "components/helpers/govuk-frontend-settings";
 @import "govuk/base";
+
+@import 'components/mixins/margins';
+// TODO: remove this focus override when govuk_template is no longer used
+@import 'components/mixins/govuk-template-link-focus-override';
+@import 'components/mixins/css3';
+
+$gem-secondary-button-colour: #00823b;
+$gem-secondary-button-hover-colour: darken($gem-secondary-button-colour, 5%);
+$gem-secondary-button-background-colour: govuk-colour("white");
+$gem-secondary-button-hover-background-colour: govuk-colour("light-grey", $legacy: "grey-4");
+
+$gem-quiet-button-colour: govuk-colour("dark-grey", $legacy: "grey-1");
+$gem-quiet-button-hover-colour: darken($gem-quiet-button-colour, 5%);
+
+$gem-hover-dark-background: #dddcdb;
```

Comment:
It feels like this line (and the button stuff above it) would be more appropriate in `component_support.scss` but I'm guessing there's a reason it's in here?

Reply:
I placed this in `govuk_frontend_support.scss` because it doesn't add anything to the compiled stylesheet, and can be used either on its own or alongside `component_support.scss`. It gives flexibility, since `govuk_frontend_support.scss` should always be there, and `component_support.scss` is optional.

I think that these two files should be renamed - but that's a breaking change, and I didn't think it necessary here.
If these files were to be renamed, I'd suggest naming them to reflect that one doesn't add anything to the compiled CSS, and one that does.

Reply:
Fair enough, sounds sensible. Yes, I also agree now that we should rename them, because they're a bit confusing. But I remember struggling to come up with good names when I created them!